.github/workflows/actions/stew-ci/action.yml

```diff
@@ -25,7 +28,7 @@ runs:
 
     - name: poetry
       shell: bash
-      run: python -m pipx install poetry==1.1.15
+      run: python -m pipx install "poetry${{ inputs.poetry-version }}"
```

Comment:
I think there needs to be `==` between `poetry` and `${{ inputs.poetry-version }}` in `"poetry${{ inputs.poetry-version }}"` no? If there are no version it's fine tho since you'd get `poetry` omitting the empty quotes

Reply:
It's unusual but I don't think it's going to be confusing, since the default value is a constraint and the description also talks about a constraint.